prow/plugins/golint/golint.go

```diff
@@ -156,12 +159,14 @@ func handle(ghc githubClient, gc *git.Client, log *logrus.Entry, ic github.Issue
 	if err := r.CheckoutPullRequest(ic.Issue.Number); err != nil {
 		return err
 	}
+	log.WithField("duration", time.Since(startClone)).Info("Cloned and checked out PR.")
```

Comment:
How long does it take for the linter to run on a typical PR? If it takes a while it might be worth logging the time spent linting in addition to the time spent cloning.

/lgtm